levels.go

```diff
@@ -37,7 +37,7 @@ import (
 
 type levelsController struct {
 	nextFileID uint64 // Atomic
-	elog       trace.EventLog
+	logger     Logger
```

Comment:
You can remove this `logger`. The `levelsController` contains `DB` struct which contains the `logger`. 

You can do `levelsController.kv.opt.Logger` where ever you need this.